Resources/Prototypes/Entities/Mobs/NPCs/pets.yml

```diff
@@ -776,6 +776,9 @@
         BaseUnshaded: aslime-_3
       Dead:
         Base: rainbow_baby_slime_dead
+  - type: Body  # DeltaV - Changes to add brain
+    prototype: Smiles  # DeltaV - Changes to add brain
+    requiredLegs: 1  # DeltaV - Changes to add brain
```

Comment:
dont need deltav comments on these 2 lines

Reply:
Even though it's an upstream file?

Reply:
the comment on the body line above is enough